### ISSUE TITLE
weather@mockturtl : too much uppercase letters in the title

### ISSUE THREAD

**Issue by InfoLibre** [linuxmint/cinnamon-spices-applets#3539]
```
 * Applet version 3.0.1
 * Cinnamon version 4.8.6
 * Mint 20.1
 * Graphics hardware *and* driver used
 * Snippet or attachment of ~/.cinnamon/glass.log
 * Snippet or attachment of ~/.xsession-errors

 ```

**Notify author of applet**
@mockturtl

**Issue**

There are uppercase letters at the beginning of each word in the tittle, it shouldn't :

![Capture d’écran du 2021-02-10 10-49-41](https://user-images.githubusercontent.com/3955499/107494061-9f797f80-6b8e-11eb-9384-677def34ba94.png)

**Comment by Gr3q** [comment]
These issues should be fixed with version 3.0.2, please close this issue when you confirmed

**Comment by Gr3q** [comment]
Scratch that, I forgot to include it in the version, I'll add it now
